src/main/java/frc/robot/config/Config.java

```diff
@@ -192,10 +192,10 @@ public static class CANID {
         // Distance between centers of right and left wheels on robot
 
         public static final SwerveDriveKinematics kSwerveDriveKinematics = new SwerveDriveKinematics(
-            new Translation2d(kWheelBase / 2, kTrackWidth / 2),
-            new Translation2d(kWheelBase / 2, -kTrackWidth / 2),   
             new Translation2d(-kWheelBase / 2, kTrackWidth / 2),
-            new Translation2d(-kWheelBase / 2, -kTrackWidth / 2));
+            new Translation2d(kWheelBase / 2, kTrackWidth / 2),   
+            new Translation2d(-kWheelBase / 2, -kTrackWidth / 2),
+            new Translation2d(kWheelBase / 2, -kTrackWidth / 2));
```

Comment:
Remove this change to SwerveDriveKinematics. Swapping around the negative and positive signs here was to temporarily allow the swervy beetle to have a different front. On the CompBot, we'll need the original order of positive and negative signs.